standard/dmilith.yaml

```diff
@@ -0,0 +1,28 @@
+accent: '#52ad70' # Accent color for UI elements
+details: darker # Whether the theme is lighter or darker.
+foreground: '#bbbbbb' # The foreground color.
+background: '#001000' # Terminal background color
+
+background_image:
```

Comment:
Could you comment this section out and also include a comment with a link to where someone could download the picture?

Similar to here: https://github.com/warpdotdev/themes/blob/main/warp_bundled/dark_city.yaml#L7

Reply:
To be clear, just the background image section not foreground and background haha.